src/icons/glyphs/antiAgingAndAcneGlyph.ts

```diff
@@ -0,0 +1,5 @@
+import { AntiAgingAndAcneGlyphSecondary } from './svgs';
+import { useIcon, IconProps } from '../../shared-components/icon';
+
+export default (props: IconProps) =>
+  useIcon(null, AntiAgingAndAcneGlyphSecondary, props);
```

Comment:
Since there is no corresponding primary icon, we pass in `null` here, and add a `null` check to the body of useIcon to return null if there is no icon.